### PR TITLE
Docs changes

### PR DESCRIPTION
**Pull request by Tiagoquix**
The changes that I made are just perfectionist changes. I just changed a text here and there.

Two changes I would like to highlight are:

- I removed the suggestions for disabling Game Mode and Full Screen Optimizations. Windows 10 has evolved over time and these changes are no longer strictly necessary. So much so that new games, such as Shadow of the Tomb Raider and Hitman 2 already feature a new full screen mode called Exclusive Full Screen, in which it was made exclusively to benefit from these Windows 10 changes.

- The removal of the "No lightwarps" lines from the medium and medium_high levels of the lighting module. These changes were made based on this commit: https://github.com/mastercomfig/mastercomfig/commit/f7ba3c450457ee7a018f0658bc226e9bc50412bd

### CODE REVIEW

**Comment by Tiagoquix** [comment]
Yes, I know that changes related to the docs should be targeted to the release branch, but I submitted the changes to the develop branch on purpose. I don't want them to go on the air right now.

File: docs/tf2/misconceptions.md [hunk 1]
```diff
@@ -27,13 +27,13 @@ These launch options may be creating an improper video mode, which is not prefer
...
-* `cl_interp 0.033` : This is a typo that was made somewhere along the way. The correct interp is `cl_interp_ratio 2; cl_interp 0.0303`. This is equal to 2 / 66.
-* `cl_interp 0.0152` : This means you will be subject to inaccurate extrapolation which is not in line with the server's history, all for getting lower visual latency on entity positions within a few milliseconds, which doesn't matter when positions cannot be substantially different within that time. Instead, make sure your entity positions are at least in line with the server's history as that is substantially more important for lag compensation and your own perception when predicting position based on velocity/trajectory.
+* `mat_shadowstate` : Does not exist.
+* `mat_parallaxmap` : Not used.
+* `rate 60000` : This is lower than the TF2 default rate of `80000`. TF2 default network settings are already on the low-end. Why would you set rate lower, especially when you're increasing packet rates? (`cl_updaterate 66;cl_cmdrate 66`)?
```
Comment (mastercoms) [inline docs/tf2/misconceptions.md]: This should remain a comma separated list. It is not a command, it is a structure within normal grammar describing two entities which increase packet rates.

**Review by mastercoms — CHANGES_REQUESTED** [review]
Please revert your changes to the launch options format, the preset and addons format, the removal of Windows tweaks, remove periods from see also.

**Comment by Tiagoquix** [comment]
> Please revert your changes to the launch options format, the preset and addons format, the removal of Windows tweaks, remove periods from see also.

Done. I also corrected the description for the shadows module in the docs.

**Review by mastercoms — APPROVED** [review]

**Pull request merged**